libexfat/io.c

```diff
@@ -165,6 +204,28 @@ struct exfat_dev* exfat_open(const char* spec, enum exfat_mode mode)
 		exfat_error("'%s' is neither a device, nor a regular file", spec);
 		return NULL;
 	}
+	dev->block_discard_alignment = 0;
+	dev->block_discard_granularity = 0;
+	dev->block_discard_max_bytes = 0;
```

Comment:
I'd rather memset(dev, 0, sizeof(struct exfat_dev)) after allocation.

Reply:
Switching to `calloc`.